torchgeo/datasets/utils.py

```diff
@@ -482,3 +482,31 @@ def rgb_to_mask(
         if isinstance(cmask, np.ndarray):
             mask[cmask.all(axis=-1)] = i
     return mask
+
+
+def percentile_normalization(
```

Comment:
Can you add tests for this in `tests/datasets/test_utils.py`? My goal is that if you only run the tests for a single file, that file should get 100% coverage.